### DISCUSSION TITLE
Connection issue for localhost for port below 1024

### DISCUSSION THREAD

**Discussion by SamYuan1990** [actions/virtual-environments#6358] [General questions] [answered]
https://github.com/SamYuan1990/Trash/actions/runs/3189919193/jobs/5204315488

I try to start a go server in my test code and use go client to access it for test.
the server I am listen on is 0.0.0.0:1000 and the request is targeting at localhost:1000 or 127.0.0.1:1000

the code passed locally, but fails on github action.

can anyone help advise?

**Comment by al-cheb** [comment, marked as answer]
Hey @SamYuan1990.
What about to use a port above 1024, like 10000?

**Reply by SamYuan1990** [reply]
ok, it works.
